.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Pull request
+
+on:
+  pull_request:
+    branches:
+      - main
+      - release-2.5
```

Comment:
Leaving both **main** and **release-2.5** here to allow triggers to run if/when **release-2.5** is renamed to **main**